incubator/nfs-pv-provider/README.md

```diff
@@ -0,0 +1,16 @@
+The documentation is a work-in-progress
+
+Visit [kingdonb/nfs-data](//github.com/kingdonb/nfs-data) for the most current
+README.
+
+To install this, try:
+
+```console
+$ helm install -n my --namespace nfs .
+```
+
+... then, go ahead and make some claims with ReadWriteMany according to the
+examples in [kingdonb/nfs-data](//github.com/kingdonb/nfs-data) to share an NFS
+path under the filesystem between pods.
+
+Your pods can do this without building or including any extra NFS client!
```

Comment:
I realize the docs are WIP, but moving the docs forward would help a lot with the review process here. If the docs present a clear vision for how this is meant to be used, it's easier to reason about what we're looking at. i.e. Imparting a conceptual understanding makes the implementation clearer to all.

Reply:
Noted.  I have an issue open to work on the README in that repo.

I actually wrote this great README and managed to overwrite it with the sample README before it made it into a commit.  Had to rage quit after that.